### PR TITLE
[ticket/17335] Fix language var issues in the extension manager

### PR DESCRIPTION
**Pull request by iMattPro**
PHPBB-17335

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/master/coding-guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB-17335

### CODE REVIEW

File: phpBB/language/en/acp/extensions.php [hunk 1]
```diff
@@ -36,14 +36,15 @@
...
 
-	'EXTENSION_ALREADY_INSTALLED'				=> 'The “%s” extension has already been installed.',
-	'EXTENSION_ALREADY_INSTALLED_MANUALLY'		=> 'The “%s” extension has already been installed manually.',
-	'EXTENSION_ALREADY_MANAGED'					=> 'The “%s” extension is already managed.',
-	'EXTENSION_CANNOT_MANAGE_FILESYSTEM_ERROR'	=> 'The “%s” extension cannot be managed because the existing files could not be removed from the filesystem.',
-	'EXTENSION_CANNOT_MANAGE_INSTALL_ERROR'		=> 'The “%s” extension could not be installed. The prior installation of this extension has been restored.',
-	'EXTENSION_MANAGED_WITH_CLEAN_ERROR'		=> 'The “%1$s” extension has been installed but an error occurred and the old files could not be removed. You might want to delete the “%2$s” files manually.',
-	'EXTENSION_MANAGED_WITH_ENABLE_ERROR'		=> 'The “%s” extension has been installed but an error occurred while enabling it.',
-	'EXTENSION_NOT_INSTALLED'					=> 'The “%s” extension is not installed.',
+	'EXTENSIONS_ALREADY_INSTALLED'				=> 'The “%s” extension has already been installed.',
```
[the remark this answers is not in the document]
Comment (iMattPro) [reply phpBB/language/en/acp/extensions.php:39]: Yes. The prefix is `EXTENSIONS_` here https://github.com/phpbb/phpbb/blob/e2ff7a7178a39b23dc1dbd310cbe781d85716f68/phpBB/config/default/container/services_extensions.yml#L29

File: phpBB/language/en/acp/extensions.php [hunk 2]
```diff
@@ -86,25 +89,25 @@
 	'EXTENSION_NAME'			=> 'Extension Name',
 	'EXTENSION_ACTIONS'			=> 'Actions',
 	'EXTENSION_OPTIONS'			=> 'Options',
-	'EXTENSION_INSTALL_HEADLINE'=> 'Installing an extension',
-	'EXTENSION_INSTALL_EXPLAIN'	=> '<ol>
+	'EXTENSION_INSTALLING_HEADLINE'=> 'Installing an extension',
+	'EXTENSION_INSTALLING_EXPLAIN'	=> '<ol>
```
[the remark this answers is not in the document]
Comment (iMattPro) [reply phpBB/language/en/acp/extensions.php:93]: Erm, how? Tried to do it with pluralization but can't.

[the remark this answers is not in the document]
Comment (rxu) [reply phpBB/language/en/acp/extensions.php:93]: It'll work like that:
```php
	'EXTENSIONS_INSTALL_EXPLAIN'	=> [
		'step_1'	=> 'Download an extension from phpBB’s extensions database',
		'step_2'	=> 'Unzip the extension and upload it to the <samp>ext/</samp> directory of your phpBB board',
		'step_3'	=> 'Enable the extension, here in the Extensions manager',
		],

```

```html
	<tr>
		<td class="row3"><ol>
			<li>{{ lang(['EXTENSIONS_INSTALL_EXPLAIN', 'step_1']) }}</li>
			<li>{{ lang(['EXTENSIONS_INSTALL_EXPLAIN', 'step_2']) }}</li>
			<li>{{ lang(['EXTENSIONS_INSTALL_EXPLAIN', 'step_3']) }}</li>
		</ol></td>
	</tr>
```

[the remark this answers is not in the document]
Comment (iMattPro) [reply phpBB/language/en/acp/extensions.php:93]: Yeah, I'd rather it be in a loop.
```twig
{% for row in EXTENSION_INSTALLING_EXPLAIN %}
    <li>{{ lang(row) }}</li>
{% endfor %}
```

[the remark this answers is not in the document]
Comment (rxu) [reply phpBB/language/en/acp/extensions.php:93]: You've broke all the phpBB lang tool magic lol.

[the remark this answers is not in the document]
Comment (rxu) [reply phpBB/language/en/acp/extensions.php]: It works other way:
```twig
{% for row in ['step_1', 'step_2', 'step_3'] %}
	<li>{{ lang(['EXTENSION_INSTALL_EXPLAIN', row]) }}</li>
{% endfor %}
```

**Comment by rxu** [comment]
I'd add `lang_raw` Twig function to iterate language array entries your way via loops.

**Pull request merged**